### PR TITLE
Port blog inside the site

### PR DESCRIPTION
**Pull request by Jean85**
I started this branch a few months ago: I wanted to test if it is viable to leave Medium and host our blog posts ourselves.

I ported the first two articles in markdown as examples, and I've created a basic blog page with a list.

I need help with:
 * fixing the template of the post: the layout works for the list page, but not for the post itself
 * add navigation bits and show metadata (timestamp, author) at the top of the post
 * a few CSS addition: after adding stuff from the previous point, I don't know how to style it

Anyone wanna help? @stefanotorresi? @goetas?

### CODE REVIEW

**Comment by goetas** [comment]
I have checked it out, will see what can I help in the next days/weeks!

**Comment by goetas** [comment]
Is part of the plan having an "author" section? Or no author will be mentioned?

**Comment by Jean85** [comment]
This PR is getting deployed here: https://www---pr-258-zvlugly-ovkfaqrdltr5s.us.platform.sh/

**Comment by stefanotorresi** [comment]
it may be worth exploring if https://github.com/calebporzio/gitdown produces a 1:1 output with Kramdown, so that we finally get rid of that Ruby nonsense!

**Comment by goetas** [comment]
Imported medium posts, some manual work still needs to be done, but 80% of it is there

**Comment by goetas** [comment]
Most of the fixes are done now

**Comment by Jean85** [comment]
And it deployed correctly! https://www---pr-258-zvlugly-ovkfaqrdltr5s.us.platform.sh/blog/ :tada: 

It looks already fine, it just needs some CSS touches and a few navigation (back?) button. Also, did we set a limit to the "recent posts" column?

**Comment by goetas** [comment]
pushed few more adjustments (including the limit on recent posts). 
Not sure where to place the back button... (in the browser maybe :D ?)

**Comment by goetas** [comment]
@stefanotorresi 

> it may be worth exploring if https://github.com/calebporzio/gitdown produces a 1:1 output with Kramdown, so that we finally get rid of that Ruby nonsense!

`calebporzio/gitdown` still uses the Github api to generate markdown, thus there is no difference with `Aptoma\Twig\Extension\MarkdownEngine\GitHubMarkdownEngine` already part of the `aptoma/twig-markdown` package. 
Working offline is a must. In this case I think that the ruby non-sense unfortunately has to stay...

File: source/_pages/blog.twig [hunk 1]
```diff
@@ -22,10 +22,12 @@ use:
             <h2 class="sidebar__title">Recent posts:</h2>
             <ul class="sidebar__list">
                 {% for recent in data.posts %}
+                    {% if loop.index < 8 %}
```
Comment (Jean85) [inline source/_pages/blog.twig]: Maybe you can `{% for recent in data.posts|slice(0,8) %}`?

Comment (goetas) [reply source/_pages/blog.twig]: done

**Comment by goetas** [comment]
@Jean85 Can you please check which CSS improvements are still required here? Would like to proceed with this to hot have it sitting for months as WIP..

**Comment by Jean85** [comment]
Sorry for the delay, I'm on vacation. I did a quick scan and everything seems good CSS-wise. The only fix that I would add is a bit of additional spacing for tags and the end of the articles in the single page, like here: https://www---pr-258-zvlugly-ovkfaqrdltr5s.us.platform.sh/blog/2019/05/updates-from-the-php-fig-up-until-the-may-elections/

image

**Comment by goetas** [comment]
@Jean85 margins increased 

image

**Comment by Jean85** [comment]
Great, LGTM now! @iansltx, did you take a look at this?

@goetas, are all posts already imported?

**Comment by iansltx** [comment]
Will have a look tomorrow.

**Comment by goetas** [comment]
> @goetas, are all posts already imported?

@Jean85 yes, all from https://medium.com/php-fig

**Comment by Jean85** [comment]
Great! If we all agree we could merge this, and take the occasion to write a new post talking about this migration and PSR-12 too.

**Comment by goetas** [comment]
@iansltx check the `app/config/sculpin_kernel_dev.yml.example` script, just copy+rename it in `app/config/sculpin_kernel_dev.yml` and you get it locally super fast (with small ui penalty)

**Comment by goetas** [comment]
@iansltx @Jean85 Any more comment on this?

**Comment by iansltx** [comment]
Everything lgtm as of the latest changes I pished.

**Comment by Jean85** [comment]
Sorry @goetas I have very limited bandwidth lately :disappointed: I'll lay down what's the intended next steps for me and we start from that:

 * I'll ping the CC and the others on Slack to let all know that we're proceeding with this: I don't want to have anyone complain AFTER we deploy this
 * We should prepare a new post to be published here (maybe another PR on top of this branch?) that announces this change, and states the PSR-12 approval that happened in the meantime
 * After we're ready, and we published all of this, we should edit the Medium pieces to cross-link to our site.

WDYT? I'll tackle the first point, I doubt I'll have time for the other two... Anyone of you two can do no. 2?

**Comment by iansltx** [comment]
I should have time this coming weekend at CascadiaPHP.

**Comment by Jean85** [comment]
Merged on develop. I'll work on a PR for the next blog post ASAP.

**Comment by goetas** [comment]
Yay

**Comment by Jean85** [comment]
PR opened: #262

**Pull request merged**